modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -95,6 +105,10 @@ export default {
   },
   mounted() {
     this.validateAndEmit();
+    apos.bus.$on('context-menu-item-clicked', item => {
```

Comment:
This seems to be an unnecessary use of the bus. `AposRelationshipFieldsManager` could be hidden with the `v-show` directive and revealed from the click event from the slat.

Reply:
Done